app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
```

Comment:
- [ Required ] Hi guys you did well working on the functionality of this App, however, I think the styling of the `login` and `registration` forms is not consistent with the project wireframe provided for this milestone. Currently, your design varies with the project wireframe in the following way. 

   - [ ] The width of the name, email, password `input` fields, and the login `button` should be the same, in your design they have varied lengths. Kindly fix this.
   - [ ] In the project wireframe the `input` fields have placeholder texts while the input fields in your design don't have placeholder texts in them. Kindly fix this.

## Your design 
![Screenshot from 2023-09-02 15-43-49](https://github.com/Odongo006/Rails_blog_App/assets/16392046/31dcb4c4-70b7-46fc-9f98-02cad17b49ec)

## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_login.png)

Reply:
- [ ] [REQUIRED] As recommended by the previous reviewers, kindly ensure that the UI for the sign-up and sign-in page looks just like the wireframe. This is a hard project requirement that must be met. Kindly implement the fixes before requesting another review.
- [ ] Kindly ensure that the width of the (name, email, password) input fields, and the login button should be the same.
- [ ] Also, the wireframe provided has placeholder texts in the input fields. Let's move the placeholder texts for your design into their input fields as well.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="349" alt="Screenshot 2023-09-02 at 18 32 40" src="https://github.com/Odongo006/Rails_blog_App/assets/58692590/f32cc6be-c0bc-428d-903a-2b623e775450"> | ![blog_login](https://github.com/Odongo006/Rails_blog_App/assets/58692590/bb9f32c9-216f-4652-b48c-5fd0a5409523)